codespell_lib/data/dictionary.txt

```diff
@@ -41126,16 +41140,27 @@ successully->successfully
 succesully->successfully
 succicently->sufficiently
 succint->succinct
+succintly->succinctly
+succseed->succeed
 succseeded->succeeded
+succseeding->succeeding
+succseeds->succeeds
 succsess->success
+succsessful->successful
 succsessfull->successful
+succsessfully->successfully
 succsessive->successive
 succssful->successful
+succssfully->successfully
+succussful->successful
 succussfully->successfully
 suceed->succeed
 suceeded->succeeded
 suceeding->succeeding
 suceeds->succeeds
+suceess->success
+suceesses->successes
```

Comment:
Aren't we missing the following? Not that there are many of these typos out there, just for consistency.
```
suceession->succession
suceessive->successive
```